lang/java/reef-tests/src/test/java/org/apache/reef/tests/yarnpreemption/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Test reef.io.watcher.
+ */
+package org.apache.reef.tests.yarnpreemption;
```

Comment:
Regarding the package naming: in `reef-tests` we have generic unit tests that should work with any runtime (and many tests actually switch e.g. between local and YARN depending on `REEF_TEST_YARN` environment variable). We try not to refer a specific runtime and expect that all tests in this package can be run in local mode.

That is, I would either move the preemption tests to `reef-runtime-yarn`, or keep them in `reef-tests` and rename the package to `org.apache.reef.tests.preemption`.